src/__tests__/patients/allergies/AllergiesList.test.tsx

```diff
@@ -16,52 +15,36 @@ describe('Allergies list', () => {
     jest.spyOn(PatientRepository, 'find').mockResolvedValueOnce(mockPatient)
     const history = createMemoryHistory()
     history.push(`/patients/${mockPatient.id}/allergies`)
-    let wrapper: any
     await act(async () => {
-      wrapper = await mount(
+      await render(
         <Router history={history}>
           <AllergiesList patientId={mockPatient.id} />
         </Router>,
       )
     })
-
-    wrapper.update()
-    return { wrapper: wrapper as ReactWrapper, history }
+    return { history }
   }
 
   it('should render a list of allergies', async () => {
     const expectedAllergies = [{ id: '456', name: 'some name' }]
-    const { wrapper } = await setup(expectedAllergies)
-    const listItems = wrapper.find(ListItem)
-
-    expect(wrapper.exists(List)).toBeTruthy()
+    await setup(expectedAllergies)
+    const listItems = screen.getAllByRole('button')
     expect(listItems).toHaveLength(expectedAllergies.length)
-    expect(listItems.at(0).text()).toEqual(expectedAllergies[0].name)
+    expect(screen.getByText(expectedAllergies[0].name)).toBeInTheDocument()
   })
 
   it('should display a warning when no allergies are present', async () => {
     const expectedAllergies: Allergy[] = []
-    const { wrapper } = await setup(expectedAllergies)
-
-    const alert = wrapper.find(Alert)
-
-    expect(wrapper.exists(Alert)).toBeTruthy()
-    expect(wrapper.exists(List)).toBeFalsy()
-
-    expect(alert.prop('color')).toEqual('warning')
-    expect(alert.prop('title')).toEqual('patient.allergies.warning.noAllergies')
-    expect(alert.prop('message')).toEqual('patient.allergies.addAllergyAbove')
+    await setup(expectedAllergies)
+    expect(screen.getByText('patient.allergies.warning.noAllergies')).toBeInTheDocument()
```

Comment:
could we use `getByRole` and look for an alert?

Reply:
✅